.github/workflows/presubmit-image.yaml

```diff
@@ -0,0 +1,24 @@
+on:
+  pull_request:
+
+jobs:
+  presubmit-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.20.x'
+      - uses: chainguard-dev/actions/melange-build@main
+        with:
+          config: melange.yaml
+          archs: x86_64
+          sign-with-temporary-key: true
+      - uses: chainguard-images/actions/apko-publish@main
```

Comment:
I see that this action pushes the image with a digest instead of a tag. Our tests currently leverages a `--kotsadm-tag` flag to override the image used during the install. For example: https://github.com/replicatedhq/kots/blob/v1.100.3/.github/workflows/build-test.yaml#L988-L990